tests/suite/test_ac_policies.py

```diff
@@ -59,6 +65,9 @@ def fin():
             ingress_controller_prerequisites.namespace,
             std_cm_src,
         )
+        file = f"reload-{get_test_file_name(request.node.fspath)}.json"
```

Comment:
I wonder if it would help to have a helper type that incapsulates the logging of reload times. Because currently the logic is repeated in multiple files, and we can incapsulate that logic in one place.  

For example, we create a new type:
```
def create_reload_time_logger(filename):
# returns ReloadTimeLogger initialized with the file name 

class ReloadTimeLogger:
   def log(time) # logs time in ms to the file 
```

And use that type in our tests. Fo example, during test setup we create the logger:
```
logger = create_reload_time_logger(filename)
```

and then we can use it in the tests:
```
logger.log(reload_time)
```

Reply:
Added a simpler function